agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.52.1"
+const Version = "1.52.2"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "3e46f380"
+const GitShortHash = "9349e80e"
```

Comment:
the new has is cc51229f3b1a5131c54ce0d36c376c99b27dfe28 after the PR [Give up trying to stop container if max timeouts are reached #2874](https://github.com/aws/amazon-ecs-agent/pull/2874) is merged : )

Reply:
i don't think this sha will be an actual commit sha because when you change this line the commit sha changes.. but anyway i updated it again

Reply:
ok! The sha cc51229 is shown under this PR's commits tab
  
<img width="1612" alt="Screen Shot 2021-05-20 at 12 38 34 PM" src="https://user-images.githubusercontent.com/69879273/119057677-9b68b500-b968-11eb-81e1-7a4acdca70fb.png">

Reply:
ok i see. updated